steamapi/user.py

```diff
@@ -36,7 +36,8 @@ def __init__(self, badge_id, level, completion_time, xp, scarcity, appid=None):
         if type(completion_time) is datetime.datetime:
             self._completion_time = completion_time
         else:
-            self._completion_time = datetime.datetime.fromtimestamp(completion_time)
+            self._completion_time = datetime.datetime.fromtimestamp(
```

Comment:
No need to break lines here, I know PEP-8 has a 80-characters-per-line rule but the (relatively) small lib doesn't adhere to that anyway. It's best to either keep it as-is or fix it all. (And for now I prefer to just make this small change, rather than PEP-8-ify it all)

Reply:
Hey @smiley. 

I have my editor configured to apply autopep8 on save. 

I have reverted the automatic formatting changes.